server/src/routers/api/userRouter.js

```diff
@@ -41,4 +48,15 @@ router.get("/", auth, (req, res) => {
   res.send(req.user);
 });
 
+// @route POST api/users/logout
+// @desc Logout a user
+// @access Private
+
+router.post("/logout", auth, async (req, res) => {
+  res.clearCookie("token");
```

Comment:
You have to invalidate the token
Meaning -> Find this token (in user's token array ) and remove it from that array. Save the changes made.